crossdock/client/gauntlet/behavior.go

```diff
@@ -379,7 +379,7 @@ func Run(t crossdock.T) {
 		}
 
 		ctx, _ := context.WithTimeout(context.Background(), time.Second)
-		req := thrift.Request{
+		req := thrift.ReqMeta{
```

Comment:
nit: `reqMeta` to match the other renames? (i'm also fine with not renaming, up to you).